xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/less/lightbox.less

```diff
@@ -52,12 +54,13 @@
 }
 .blueimp-gallery {
   .action:hover {
-    color: @text-color;
+    color: @panel-default-text;
     opacity: 1;
   }
 
   .action {
-    color: @xwiki-page-content-bg;
+    color: @panel-default-text;
+    opacity: 0.8;
```

Comment:
I thought it was a bad idea to use opacity with colors?

Reply:
I chose to use an opacity of 0.8 here since we don't have a variation of `@text-muted` that would match the color in `@panel-default-text` (something like `@panel-muted-text`). See [this ticket](https://jira.xwiki.org/browse/XWIKI-21105) and [this topic on the forum](https://forum.xwiki.org/t/color-theme-panel-content/12818) for a discussion about the state of panel colours in the colorthemes.

Opacity can reduce the importance of a text on both light and dark theme, which is not true for the LESS color operation functions lighten/darken.

This is not a perfect solution, but I think this is the one with the least drawbacks.